website/sidebars.json

```
@@ -69,6 +70,7 @@
           "context/ref/Country",
           "context/ref/Currency",
           "context/ref/Email",
+          "context/ref/Interaction",
           "context/ref/Instrument",
           "context/ref/InstrumentList",
```

Alphabetical order pls
```suggestion
          "context/ref/Instrument",
          "context/ref/InstrumentList",
          "context/ref/Interaction",
```